src/ResourceManager/CognitiveServices/CognitiveServices.Test/ScenarioTests/CognitiveServicesAccountTests.ps1

```diff
@@ -0,0 +1,391 @@
+
+        Assert-AreEqual $updatedKeys.Key1 $reupdatedKeys.Key1;
+        Assert-AreNotEqual $originalKeys.Key2 $reupdatedKeys.Key2;
+
+        Retry-IfException { Remove-AzureRmCognitiveServicesAccount -ResourceGroupName $rgname -Name $accountname; }
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $rgname
+    }
+}
+
+
+<#
+.SYNOPSIS
+Test Get-AzureRmCognitiveServicesAccountSkus
+#>
+function Test-GetAzureRmCognitiveServicesAccountSkus
+{
+    # Setup
+    $rgname = Get-CognitiveServicesManagementTestResourceName;
+    
+    try
+    {
+        # Test
+        $accountname = 'csa' + $rgname;
+        $skuname = 'S2';
+        $accounttype = 'TextAnalytics';
+        $loc = 'West US';
+
+        New-AzureRmResourceGroup -Name $rgname -Location $loc;
+        New-AzureRmCognitiveServicesAccount -ResourceGroupName $rgname -Name $accountname -Type $accounttype -SkuName $skuname -Location $loc;
+        
+        $skus = (Get-AzureRmCognitiveServicesAccountSkus -ResourceGroupName $rgname -Name $accountname).Value | Select-Object -ExpandProperty Sku;
+        $skuNames = $skus | Select-Object -ExpandProperty Name
+        
+        $expectedSkus = "F0", "S1", "S2", "S3", "S4"
+        Assert-AreEqualArray $expectedSkus $skuNames
+
+        Retry-IfException { Remove-AzureRmCognitiveServicesAccount -ResourceGroupName $rgname -Name $accountname; }
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $rgname
+    }
+}
+
+<#
+.SYNOPSIS
+Test Get-AzureRmCognitiveServicesAccount | Get-AzureRmCognitiveServicesAccountKey
+#>
+function Test-PipingGetAccountToGetKey
+{
+    # Setup
+    $rgname = Get-CognitiveServicesManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $accountname = 'csa' + $rgname;
+        $skuname = 'S2';
+        $accounttype = 'TextAnalytics';
+        $loc = 'West US';
+
+        New-AzureRmResourceGroup -Name $rgname -Location $loc;
+        New-AzureRmCognitiveServicesAccount -ResourceGroupName $rgname -Name $accountname -Type $accounttype -SkuName $skuname -Location $loc;
+
+        $keys = Get-AzureRmCognitiveServicesAccount -ResourceGroupName $rgname -Name $accountname | Get-AzureRmCognitiveServicesAccountKey;
+        Assert-AreNotEqual $keys.Key1 $keys.Key2;
+
+        Retry-IfException { Remove-AzureRmCognitiveServicesAccount -ResourceGroupName $rgname -Name $accountname; }
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $rgname
+    }
+}
+
+<#
+.SYNOPSIS
+Test Get-AzureRmCognitiveServicesAccount | Set-AzureRmCognitiveServicesAccount
+#>
+function Test-PipingToSetAzureAccount
+{
+ # Setup
+    $rgname = Get-CognitiveServicesManagementTestResourceName
+
+    try
+    {
+        # Test
+        $accountname = 'csa' + $rgname;
+        $skuname = 'S2';
+        $accounttype = 'TextAnalytics';
+        $loc = 'West US';
```

Comment:
commented about location.
why the linebreaks are so important? the code will not look nice with them, IMHO.